website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,295 @@
+- Existing users with an email address in the federated domain will automatically be converted to federated authentication, effectively _taking ownership_ of the account.
+- User accounts with the role of Administrator, Site Manager, or People Manager can’t sign in using federated authentication; they can only manage the federation process.
+
+:::
+
+### Placeholders
+
+The following placeholders are used:
+
+- `authentik.company`: The Fully Qualified Domain Name of the authentik installation.
+
+## authentik Configuration
+
+Before you can proceed with the Apple Business Manager configuration, you'll need to configure authentik to act as an identity provider for Apple. Start by logging into your authentik instance and navigating to the Admin interface.
+
+### Scope Mapping
+
+In the Admin interface, navigate to **Customization** -> **Property Mappings**.
+
+Press **Create** and when prompted for a choice of mapping, select **Scope Mapping**.
+
+#### Apple Business Manager Profile
+
+Create a new scope mapping to handle the user's profile information:
+
+| Field      | Value                          |
+| ---------- | ------------------------------ |
+| Name       | Apple Business Manager profile |
+| Scope name | `profile`                      |
+
+Use the following expression to map the user's profile information:
+
+```py title="Expression"
+first_name, _, last_name = request.user.name.partition(" ")
+
+return {
+  "name": request.user.name,
+  "given_name": first_name,
+  "family_name": last_name,
+  "preferred_username": request.user.username,
+  "nickname": request.user.username,
+  "groups": [group.name for group in request.user.ak_groups.all()],
+}
+```
+
+Save the scope mapping and press "Create" to continue to the next scope mapping.
+
+#### Read Scope
+
+Next, create a new scope mapping to handle read access to the user's profile information:
+
+| Field      | Value                           |
+| ---------- | ------------------------------- |
+| Name       | Apple Business Manager ssf.read |
+| Scope name | ssf.read                        |
+| Expression | `return {}`                     |
+
+Once again, save the scope mapping and press "Create" to continue to create the final scope mapping.
+
+#### Manage Scope
+
+| Field      | Value                             |
+| ---------- | --------------------------------- |
+| Name       | Apple Business Manager ssf.manage |
+| Scope name | ssf.manage                        |
+| Expression | `return {}`                       |
+
+Finally, save the scope mapping and visually confirm the creation of all three scope mappings.
+
+### OAuth2/OpenID Provider
+
+From the authentik Admin interface, navigate to **Applications** -> **Providers** and press **Create**.
+
+:::tip{title="Keep your text editor ready"}
+
+authentik will automatically generate the _Client ID_ and _Client Secret_ values for the new provider.
```

Comment:
```suggestion
authentik will automatically generate the `Client ID`  and `Client Secret` values for the new provider.
```

Reply:
We want values to be in code font, not italics. Italics are reserved for variables. ;-)

Reply:
Also need to have numbered steps, please